client/tutorials/ibm-blockchain-platform-vscode-smart-contract/index.md

```diff
@@ -1,520 +1,230 @@
 
-primary_tag: "blockchain"
+1. In the left sidebar, click on the __IBM Blockchain Platform__ icon (it looks like a square, and will probably be at the bottom of the set of icons if this was the latest extension you installed!)
 
-pta:
- - "emerging technology and industry"
+2. Mouse-over the `SMART CONTRACT PACKAGES` panel, click the `...` menu, and select `Create Smart Contract Project` from the dropdown.
 
-pwg:
-  - "blockchain"
+> Command Palette alternative: `IBM Blockchain Platform: Create Smart Contract Project`
 
-related_content:
-  - type: announcements
-    slug: "ibm-blockchain-platform-vscode-smart-contract"
-  - type: tutorials
-    slug: "cl-ibm-blockchain-101-quick-start-guide-for-developers-bluemix-trs"
-  - type: patterns
-    slug: "create-and-execute-blockchain-smart-contracts"
+3. Choose a smart contract language. JavaScript, TypeScript, Java and Go are all available. For the purpose of this tutorial, please choose `TypeScript`.
 
-related_links:
-  - title: "Video: Start developing with the IBM Blockchain Platform VSCode Extension"
-    url: "https://youtu.be/0NkGGIUPhqk"
-  - title: "IBM Blockchain Platform"
-    url: "https://www.ibm.com/blockchain/platform"
-  - title: "IBM Blockchain - Marbles demo"
-    url: "https://github.com/IBM-Blockchain/marbles"
-  - title: "Hyperledger Fabric docs"
-    url: "https://hyperledger-fabric.readthedocs.io/en/release-1.2/"
+4. The extension will ask you if you want to name the asset in the generated contract. This will default to `MyAsset`, but you're welcome to have some fun ;)  What do you intend to use your blockchain for? This will determine what type of asset you create, update and read from the ledger: `Radish`? `Pineapple`? `Penguin`? Pick whatever you like! For the sake of this tutorial, we'll be boring and stick with `MyAsset`.
 
-# runtimes:
+> __Pro Tip:__ If you decide to change the name of your asset, remember to swap out `MyAsset` for whatever you named it in future steps!
 
-services:
-  - "blockchain"
+5. Choose a location to save the project.  Click `Browse`, them click `New Folder`, and name the project what you want e.g. `demoContract`.
 
-subtitle: "Streamline the process of developing, testing, and deploying a smart contract"
+6. Click `Create` and then select the new folder you just created and click `Save`.
 
-tags:
-  - "blockchain"
+7. Finally, select `Add to workspace` from the list of options.
 
-title: "Develop a smart contract with the IBM Blockchain Platform VSCode extension"
+The extension will generate you a skeleton contract based on your selected language and asset name. Once it's done, you can navigate to the __Explorer__ view (most-likely the top icon in the left sidebar, which looks like a "document" icon) and open the `src/my-asset-contract.ts` file to see your smart contract code scaffold. Great work, you've got yourself a smart contract - let's take a look at its contents...
```

Comment:
Is this not just `src/my-contract.ts`? - https://github.com/IBM-Blockchain/generator-fabric/tree/master/generators/contract/templates/typescript/src